solara/lab/components/datepicker.py

```diff
@@ -0,0 +1,150 @@
+import datetime as dt
+from typing import List, Optional, Union
+
+import solara
+import solara.lab
+
+
+@solara.component
+def DatePicker(
+    date: solara.Reactive[dt.date],
+    children: Optional[List[solara.Element]] = [],
+    open: Optional[Union[solara.Reactive[bool], bool]] = False,
+    date_format: str = "%Y-%m-%d",
+):
```

Comment:
I think this should be named an 'InputDate' similar to the InputInt/Float etc. I think the v-date-picker in vue is maybe not a good name. Looking at MUI https://mui.com/x/react-date-pickers/date-picker/ and https://mui.com/x/react-date-pickers/date-calendar/ it seems like MUI also uses the name 'DatePicker' for what you are making here.
Maybe a better set of names is 'DateCalender' or 'Calendar' and 'InputDate' (we can argue about naming it DateInput later).

In any case, lets not yet expose the Calendar component, and keep that for later.

Reply:
`InputDate` and `InputDateRange` it is.